docs/man/ctags-faq.7.rst

```diff
@@ -0,0 +1,391 @@
+.. _ctags-faq(7):
+
+==============================================================
+ctags-faq
+==============================================================
+--------------------------------------------------------------
+Universal Ctags FAQ
+--------------------------------------------------------------
+:Version: 5.9.0
+:Manual group: Universal Ctags
+:Manual section: 7
+
+This is the Universal Ctags FAQ (Frequently-Asked Questions).
+It is based on `Exuberant Ctags FAQ <http://ctags.sourceforge.net/faq.html>`_
+
+.. contents::
+
+DESCRIPTION
+-----------
+
+.. TODO: https://github.com/universal-ctags/ctags/issues/2312
+	#1421: feature: clean up stale tags when appending (`-a`)
+	#2356: can't pre-process the macro but it works with Exuberant Ctags 5.8
+	#2540: C/C++：conditional compilation like #ifdef will cause parse errror
+
+What are these strange bits of text beginning with ``;"`` which follow many of the lines in the tag file?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+These are *extension flags*. They are added in order to provide extra
+information about the tag that may be utilized by the editor in order to
+more intelligently handle tags. They are appended to the EX command part of
+the tag line in a manner that provides backwards compatibility with existing
+implementations of the Vi editor. The semicolon is an EX command separator
+and the double quote begins an EX comment. Thus, the extension flags appear
+as an EX comment and should be ignored by the editor when it processes the
+EX command.
+
+Some non-vi editors, however, implement only the bare minimum of EX commands
+in order to process the search command or line number in the third field of
+the tag file. If you encounter this problem, use the option ``--format=1`` to
+generate a tag file without these extensions (remember that you can set the
+CTAGS environment variable to any default arguments you wish to supply). Then
+ask the supplier of your editor to implement handling of this feature of EX
+commands.
+
+Why can't I jump to ``class::member``?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Because, by default, ctags only generates tags for the separate identifiers
+found in the source files. If you specify the ``--extra=+q`` option, then
```

Comment:
``--extras=+q`` instead of ``--extra=+q`` in u-ctags.

Reply:
Ditto.
Fixed.